ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -34,6 +35,30 @@ Create Pipeline Server
     Wait Until Generic Modal Disappears
     Wait Until Project Is Open    project_title=${project_title}
     ...    timeout-pre-spinner=5s    timeout-spinner=60s
+    # Check if there is any CA Bundle
+    # I did not find a better way to check if multi-line string is empty or "NONE"
```

Comment:
Can you try 
`if  "${aws_s3_ca_bundle}"!="${SPACE}"`

Reply:
The problem with simple comparing multi-line strings (certs are multi-line) like '{aws_s3_ca_bundle}' == '' is
```
Invalid IF condition: Evaluating expression ''-----BEGIN CERTIFICATE-----
MIIFYTCCA0mgAwIBAgIURaJI92kpgDRwDdVeLN9DkaVSo+YwDQYJKoZIhvcNAQEL
...
80dd2sLkoUcy2tcoIs7+N9g9jykfSVNeAGH2DSHO6BNLeMZHSQ==
-----END CERTIFICATE-----
' == ''' failed: SyntaxError: EOL while scanning string literal (<string>, line 1)

Variables in the original expression ''${aws_s3_ca_bundle}' == ''' were resolved before the expression was evaluated. Try using ''$aws_s3_ca_bundle' == ''' syntax to avoid that. See Evaluating Expressions appendix in Robot Framework User Guide for more details.
```
thus, I added the comparison via string length.

Reply:
Can you show the exact syntax that caused `Invalid IF condition: Evaluating expression` ?

Wrong spacing can cause it (https://stackoverflow.com/questions/72140925/how-to-use-if-and-else-in-robot-framework)

Reply:
Hi @manosnoam it is not related `IF` condition in my opinion. The problem is the multi-line string

```
Invalid IF condition: Evaluating expression '"-----BEGIN CERTIFICATE-----
MIIFYTCCA0mgAwIBAgIURaJI92kpgDRwDdVeLN9DkaVSo+YwDQYJKoZIhvcNAQEL
BQAwKjEoMCYGA1UEAwwfc3RvcmFnZS5kaXMtYXAwNi5vc3Aucmgtb2RzLmNvbTAe
Fw0yNDAxMTkwODQxNDlaFw0yNTAxMTgwODQxNDlaMCoxKDAmBgNVBAMMH3N0b3Jh
Z2UuZGlzLWFwMDYub3NwLnJoLW9kcy5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4IC
DwAwggIKAoICAQC8/rRGfhX0KvrxZH4jdZ1Q2ntzq5nv9m0Jzws8xZGlGsHs7eYc
02cUQIT4K+gY02ea4suM4SSQ5FYpDiHrHyo15ycHhFK/0bazZhd29TlhbebjuTs+
9xIgVaOKeFg3VSuPnVssJnDtzh7QIOhVIacHVD9mXMAYZdFs7Kt/xO+ASOrVXk3x
YrqiOmAnxvCDfahsqqNNnAa9NoLF+5nBcV89RDAV2DBvOWUZt7Tss8TgWA8oaZF5
cVB/un8goQASruqIyLA7k0WNA6sj35kHYwA5ESXICMGEu+zvRaiyCMU7oyPnmopq
O8RDcmO3iD8kbSfOrEM1umCx55RRsr8mXq3Fts+djJrspidmTyMBV/XxX9xtdzZk
CU2lGG8yehRGn7RN4nTXZNrB3ezQam1VCyoPfLWwz1VNon6O1+5wqb4C/3VsH8b7
7dl6wROSYjgXEiNFdxPsfVJMEsfjIne4iGu01Nbnl0hvZfJmY4a0YcSICkzQs2RY
3Q0he+O07IFO9qVizYZX2t/aXcxgwMbAEU9Yhwn2rfOar5dC53OpGcWWmsk0aez0
GDpJ5kIRMgkCiDnBKDHSrEphCsSukZ7OZenfhuvzUK6FOpNNTWrIcGBUt9E+Ngg+
nJNAGJyeNwd3peykSonNVDmwo9J4b4ScvtQss9NWfEs5AhZvouOEkz8U/QIDAQAB
o38wfTAdBgNVHQ4EFgQUsZcd8QFRIFoih1BwOIwKxSo4+y8wHwYDVR0jBBgwFoAU
sZcd8QFRIFoih1BwOIwKxSo4+y8wDwYDVR0TAQH/BAUwAwEB/zAqBgNVHREEIzAh
gh9zdG9yYWdlLmRpcy1hcDA2Lm9zcC5yaC1vZHMuY29tMA0GCSqGSIb3DQEBCwUA
A4ICAQAjdh3GCkT1+f74zLt83XTyc6o51BZxn+fnbw+lj6/RZbMw2NiDDCnOCaQh
    [ Message content over the limit has been removed. ]
A4ICAQAjdh3GCkT1+f74zLt83XTyc6o51BZxn+fnbw+lj6/RZbMw2NiDDCnOCaQh
iEAfrvXHqm8/8lvCHQkbevmwKwDGjDUThf5TwihtcpsOzumv1XIQCQdE6sefXxaq
3v3MfSlCIYaKASy585V7i46eOdDUwVQP51j8em9zxGhpmLmwQZ/wCK6chmuT60wP
C5+YzDL5UfvPCfv5ehSa8bX9z1OEIICqssOYIS1EXIM1ITo0hN6+8FC1AnsMMEaj
lBLbmMdg6uaUBu618YUA3k3d68UrE4yEmxdcsINF0ZLDMbM7IEvVoDKFJ0P73L4E
0quLaPJYtowOIoo4us6CZZJq54iSAVzGAe6U/xSd7agxjRQk4/MGZaw4LSZi0Kx2
bciQznyMcKjAzYoNjzLolzKOMErZq9IRrrJRqqDYRj2AP80MhHJj9bBz9DsG4kBY
MQG/PRSh/KDDssvPakdPe+jS0TAmT/LXGFUt6iC6nM5Ds2j9UcpSbMMoB6LaYT+D
rG209hdNAVN/VG/vOzeNdG9x9Ag89PlhoEfOgA06lO8ubd/uXByJPDQlptlMAPtz
G/K3mC9xReelgofM68gU51P7GwQqmDYsE9/kbS1tih4yn/ehwKPJi0GTXIDSeDP3
80dd2sLkoUcy2tcoIs7+N9g9jykfSVNeAGH2DSHO6BNLeMZHSQ==
-----END CERTIFICATE-----
" == " "' failed: SyntaxError: EOL while scanning string literal (<string>, line 1)

Variables in the original expression '"${aws_s3_ca_bundle}" == "${EMPTY}" or "${aws_s3_ca_bundle}" == "${SPACE}"' were resolved before the expression was evaluated. Try using '"$aws_s3_ca_bundle" == "$EMPTY" or "$aws_s3_ca_bundle" == "$SPACE"' syntax to avoid that. See Evaluating Expressions appendix in Robot Framework User Guide for more details.
```

Reply:
Please notice this part
```
 Evaluating expression '"-----BEGIN CERTIFICATE-----
MIIFYTCCA0mgAwIBAgIURaJI92kpgDRwDdVeLN9DkaVSo+YwDQYJKoZIhvcNAQEL
...
80dd2sLkoUcy2tcoIs7+N9g9jykfSVNeAGH2DSHO6BNLeMZHSQ==
-----END CERTIFICATE-----
" == " "' failed: SyntaxError: EOL while scanning string literal (<string>, line 1)
```

Reply:
Could you you create a dedicated keyword in Common.robot, for example `Is Multi-line String Empty` , returning a boolean?

Reply:
I have added `is_string_empty` to `Helpers.py`. This method covers common strings as well ass multi-line strings